reference/git_github.rst

```diff
@@ -3,6 +3,17 @@ Git and Github
 
 This document describes some best practices for using Git and Github.
 
+Best practises
+--------------
+- Create a separate pull request for each bug.
+- If you have multiple commits in one pull request, squash them into one. For complex patches, having multiple commits might make the review easier.
+- If you are working with multiple bugs at a time, make sure you create a separate branches for each bug.
+- If you need to make changes to the commit, perhaps after feedback, amend the original commit and comment in the pull request what you've changed.  Don't create a new pull request or commit.
+- Include  the bug number in your commit & pull request title. If possible  follow this style -`Some text description (bug XXXXXXX)`.
```

Comment:
That makes sense for bugzilla bugs, for github issues, maybe not. For example there is an internal way in github of referencing an issue in a pull request.